crates/gpui/src/platform/app_menu.rs

```diff
@@ -6,7 +6,7 @@ use util::ResultExt;
 /// A menu of the application, either a main menu or a submenu
 pub struct Menu<'a> {
     /// The name of the menu
-    pub name: &'a str,
+    pub name: Cow<'a, str>,
```

Comment:
Have we considered using a `SharedString` here?

That type is more commonly used in GPUI.

Reply:
`SharedString` is ok, it can handle same things.